deployment/hosts

```diff
@@ -1,2 +1,8 @@
-injector ansible_host=<injector_host> ansible_user=<injector-user>
 upstream ansible_host=<upstream-host> ansible_user=<upstream-user>
+
+[hyperfoil_controller]
+<injector-controller-host> ansible_hostname=<injector-controller-host> ansible_host=<injector-controller-host>
```

Comment:
why not `ansible_host` and `ansible_user`??

Reply:
I've not needed the ansible_user for the playbooks to work.

Reply:
not you. And others?

Reply:
Have added the ansible_user as an inventory attribute.